config/locales/de.yml

```diff
@@ -69,6 +69,7 @@ de:
               löschen oder das System die Diskrepanz ausbalancieren lassen.
     timer:
       start: Start
+      continue_last_session: Anschliessend Starten
```

Comment:
Session fortsetzen?

Reply:
I don't care 😆 Josua called it        continue_last_session: Anschliessend Starten